feat(warp): new pools page

dimakorzhovnik opened the pull request.

dimakorzhovnik committed:
fix(warp): update pool list

dimakorzhovnik committed:
fix(package): added packageManager

mastercyb reviewed (changes requested):
1 table, please + cap

dimakorzhovnik committed:
fix(warp): added cap to warp

- cache marketData
- create  app component

dimakorzhovnik committed:
fix(warp): update warp pages

- added pools info to indicator
- added cap  and proce to pool card
- added sort pools  by cap

dimakorzhovnik committed:
fix(warp): move pool cap to title

dimakorzhovnik committed:
fix(warp): fix procent

dimakorzhovnik committed:
fix(warp): refactoring pools pages

dimakorzhovnik committed:
fix(ipfs): move set up ipfs to app

dimakorzhovnik committed:
fix(warp): change text in info pool

dimakorzhovnik committed:
Merge branch 'master' of https://github.com/cybercongress/dot-cyber into pools-pages

dimakorzhovnik committed:
fix(warp): fix after merge

dimakorzhovnik committed:
fix(warp): update grid

-update title

dimakorzhovnik committed:
fix(app): update menu
